test/test_helpers/sphinx.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ThinkingSphinx
+  class Test
+
+    class << self
+
+      def real_time_run
+        init
+        start index: false
+        yield
+      ensure
+        stop
+      end
+      alias_method :rt_run, :real_time_run
```

Comment:
I'd prefer not having the abbreviated name, if you don't mind.

Actually, I'd even propose breaking the API by doing the following this:

```ruby
  def run(realtime: false, &block)
    return super(&block) unless realtime
    realtime_run(&block)
  end

  protected

  def realtime_run
    init
    start index: false
    yield
  ensure
    stop
  end
```

I will understand of course if you prefer not to break ThinkingSphinx original API, in which case you can just ignore this suggestion.

Reply:
Let be good citizen ;) Adding new methods are ok, in fact we should not even add it to ThinkingSphinx::Test but have our own class ... Can be done later as refactoring (imagine using another indexer than Sphinx like ElasticSearch or Postgres)